parquet-hadoop/src/main/java/org/apache/parquet/hadoop/rewrite/ParquetRewriter.java

```diff
@@ -737,6 +737,7 @@ private void processChunk(
               encryptColumn,
               dataEncryptor,
               dataPageAAD);
+          boolean compressed = compressor != null;
```

Comment:
Shouldn't we get it from `headerV2` to keep it unchanged?

Reply:
Changed `headerV2.is_compressed` since `UNCOMPRESSED` is still a compression codec.